ui/app/templates/csi/plugins/plugin/index.hbs

```diff
@@ -0,0 +1,191 @@
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    {{/if}}
+    <div class="column">
+      <div class="boxed-section" data-test-plugin-node-availability>
+        <div class="boxed-section-head is-hollow">Node Health</div>
+        <div class="boxed-section-body">
+          <div class="columns is-centered is-bottom-aligned {{unless model.controllerRequired "is-max-half"}}">
+            <div class="column is-half">
+              {{gauge-chart
+                label="Availability"
+                value=model.nodesHealthy
+                total=model.nodesExpected}}
+            </div>
+            <div class="column">
+              <div class="metric">
+                <h3 class="label">Available</h3>
+                <p class="value">{{model.nodesHealthy}}</p>
+              </div>
+            </div>
+            <div class="column">
+              <div class="metric">
+                <h3 class="label">Expected</h3>
+                <p class="value">{{model.nodesExpected}}</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  {{#if model.controllerRequired}}
+    <div class="boxed-section" data-test-controller-allocations>
+      <div class="boxed-section-head">
+        Controller Allocations
+      </div>
+      <div class="boxed-section-body {{if model.controllers "is-full-bleed"}}">
+        {{#if model.controllers}}
+          {{#list-table
+            source=(take 10 sortedControllers)
+            class="with-foot" as |t|}}
+            {{#t.head}}
+              <th class="is-narrow"></th>
+              <td>ID</td>
+              <th>Created</th>
+              <th>Modified</th>
+              <th>Health</th>
+              <th>Client</th>
+              <th>Job</th>
+              <th>Version</th>
+              <th>Volumes</th>
+              <th>CPU</th>
+              <th>Memory</th>
+            {{/t.head}}
+            {{#t.body key="model.allocID" as |row|}}
+              {{plugin-allocation-row
+                data-test-controller-allocation=row.model.allocID
+                pluginAllocation=row.model}}
+            {{/t.body}}
+          {{/list-table}}
+        {{else}}
+          <div class="empty-message" data-test-empty-controller-allocations>
+            <h3 class="empty-message-headline" data-test-empty-controller-allocations-headline>No Controller Plugin Allocations</h3>
+            <p class="empty-message-body" data-test-empty-controller-allocations-message>No allocations are providing controller plugin service.</p>
+          </div>
+        {{/if}}
+      </div>
+      {{#if model.controllers}}
+        <div class="boxed-section-foot">
+          <p class="pull-right">
+          {{#link-to "csi.plugins.plugin.allocations"
+            model
+            data-test-go-to-controller-allocations
+            (query-params qpType=(qp-serialize (array "controller")))}}
```

Comment:
I’m not sure whether this is a true bug but I wonder about the persistence of these (and the below) query parameters which sticks around even when you choose the subnav link. Do you think the subnav should explicitly clear the query parameters maybe? 🤔
![allocation-filter-persistence](https://user-images.githubusercontent.com/43280/82449906-9204ee00-9a71-11ea-83d8-812b00ee8a54.gif)

Reply:
Yeah, I agree they should be reset. Dunno why Ember persists query params by default. I swear I want to reset them more often than not.